physics/README.md

```diff
@@ -0,0 +1,44 @@
+# Physics
+This discipline of science is quite different and even nascent with regards to the lab. However, we are open to contributions and collaborations.
+
+## Special Relativity
+The below mentioned resources help one understand the special theory of relativity, from scratch and do not really require any pre-requisites.
+
+### Course(s)
+#### [Advance Course on STR by Dr. H. C. Verma](https://www.youtube.com/playlist?list=PL8g67naApM8i21RucyrHYqyVwmurwA1UC)
+Unlike what the the name suggests, the course is a rather gentle and basic introduction to the theory of special relativity. The simplicity of the course is mainly due to the face that, the special theory of relativity itself, is a simple yet elegant theory.
+
+### Book(s)
+* [A Primer of Special Relativity by P. L. Sardesai](books/A%20Primer%20of%20Special%20Relativity.pdf)
+
+
+## Quantum Mechanics 
+
+This subject is the stepping stone to an entire flight of steps towards almost every modern science, right from Quantum Chromodynamics to Quantum Computing. However, the resources provided below, do not demand any pre-requisites as such.
+
+
+### Course(s)
+#### [Quantum Physics I by Dr. Barton Zwiebach](https://ocw.mit.edu/courses/physics/8-04-quantum-physics-i-spring-2016/)
+This courses focuses on delivering a scientifically pure approach towards Quantum Mechanics which can later be manifested and utilized in understanding and working on Quantum Computing. This course covers everything from the very grass-root level of concepts and helps develop a pretty concrete understanding and intuition for almost all aspects of modern physics backed by Quantum Mechanics. It is presented by MIT OCW and the instructor in charge is Dr. Barton Zweibach.
+
+### Book(s)
+ * [Quantum Mechanic by D. J. Griffiths](books/Quantum%20Mehcanics%20by%20D.%20J.%20Griffiths.pdf)
+
+## Relativistic Quantum Mechanics
+This is an advanced subject and is a rather mystic and unexplored(relatively) field of physics. Further, the resources too, can prove to be demotivating to a beginner without proper pre-requisites.
+
+### Pre-requisites
+* [Tensor Calculus](https://www.youtube.com/playlist?list=PLSuQRd4LfSUTmb_7IK7kAzxJtU2tpmEd3)
+* [Special Relativity](#special-relativity)
+* [Quantum Mechanics](#quantum-mechanics)
+
+### Course(s)
+[Relativistic Quantum Mechanics by Dr. Apoorva D Patel](https://www.youtube.com/playlist?list=PLbMVogVj5nJTDMhThY9xu2Tvg0u1RPuxO)
+This course is quite fast and requires a strikingly good hold over the above mentioned pre-requisites. The concepts too, are not given much physical emphasis at times and that can not really be blamed on the course but rather, is an obvious consequence of the nature of the subject. The book by Dr. Greiner is a gentle and friendlier resource.
```

Comment:
"times and that ~can not~ cannot really", maybe :sweat_smile:

Reply:
Oh yea sorry